test/api_tests.jl

```diff
@@ -196,16 +212,16 @@ end
     end
 
     space = GridSpace((10, 10))
-    model = ABM(Union{Daisy,Land}, space; warn = false)
+    model = ABM(Union{Daisy,Land}, space; warn=false)
```

Comment:
Please make this PR easy to review by commiting changes like these, i.e., syntactic style. We will apply an automated code linter in future PRs but until then no reason to spend dev effort reviewing linting changes.

Reply:
Additionally, I think we're fairly consistent with including spaces around `=` in keyword arguments in the rest of the repository, so this change is making parts of the code inconsistent with everything else.

Reply:
Do you have a JuliaFormatter.toml with the repo's preferred style?

Reply:
Linting changes un-committed, new issue RE JuliaFormatter